### ISSUE TITLE
Pods Pages setting not saving

### ISSUE THREAD

**Issue by MasterOfWeb** [pods-framework/pods#6942]
### Description

When I create a a Pods Page, nothing but the "enter URL here" field is saved.

There are no errors in the UI or the browser conole.

Tried on a fresh WP install. Same on a hosted site and on local

### Version

2.9.8

### Testing Instructions

1. go to /wp-admin/edit.php?post_type=_pods_page
2. fill out fields
3. click save
4. everything is gone

### Screenshots / Screencast

[/wp-admin/edit.php?post_type=_pods_page](https://share.getcloudapp.com/d5unJ505)

### Site Health Information

```text
`
### wp-core ###

version: 6.0.2
site_language: de_DE
user_language: de_DE
timezone: +00:00
permalink: /%postname%/
https_status: false
multisite: false
user_registration: 0
blog_public: 1
default_comment_status: open
environment_type: local
user_count: 1
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /Users/jangruchow/Local Sites/gamingpods/app/public
wordpress_size: 49,98 MB (52404874 bytes)
uploads_path: /Users/jangruchow/Local Sites/gamingpods/app/public/wp-content/uploads
uploads_size: 0,00 B (0 bytes)
themes_path: /Users/jangruchow/Local Sites/gamingpods/app/public/wp-content/themes
themes_size: 14,74 MB (15459118 bytes)
plugins_path: /Users/jangruchow/Local Sites/gamingpods/app/public/wp-content/plugins
plugins_size: 12,21 MB (12802991 bytes)
database_size: 3,19 MB (3342336 bytes)
total_size: 80,12 MB (84009319 bytes)

### wp-active-theme ###

name: Twenty Twenty (twentytwenty)
version: 2.0
author: WordPress-Team
author_website: https://de.wordpress.org/
parent_theme: none
theme_features: core-block-patterns, widgets-block-editor, automatic-feed-links, custom-background, post-thumbnails, custom-logo, title-tag, html5, align-wide, responsive-embeds, customize-selective-refresh-widgets, editor-color-palette, editor-font-sizes, editor-styles, widgets, menus, editor-style
theme_path: /Users/jangruchow/Local Sites/gamingpods/app/public/wp-content/themes/twentytwenty
auto_update: Deaktiviert

### wp-themes-inactive (4) ###

bootScore Child: version: 5.2.0.0, author: bootScore, Automatische Aktualisierungen deaktiviert
bootScore: version: 5.2.1.1, author: bootScore 5, Automatische Aktualisierungen deaktiviert
Twenty Twenty-One: version: 1.6, author: WordPress-Team, Automatische Aktualisierungen deaktiviert
Twenty Twenty-Two: version: 1.2, author: WordPress-Team, Automatische Aktualisierungen deaktiviert

### wp-plugins-active (1) ###

Pods - Custom Content Types and Fields: version: 2.9.8, author: Pods Framework Team, Automatische Aktualisierungen deaktiviert

### wp-media ###

image_editor: WP_Image_Editor_GD
imagick_module_version: Nicht verfügbar
imagemagick_version: Nicht verfügbar
imagick_version: Nicht verfügbar
file_uploads: File uploads is turned off
post_max_size: 1000M
upload_max_filesize: 300M
max_effective_size: 300 MB
max_file_uploads: 20
gd_version: bundled (2.1.0 compatible)
gd_formats: GIF, JPEG, PNG, WebP, BMP
ghostscript_version: not available

### wp-server ###

server_architecture: Darwin 21.6.0 x86_64
httpd_software: Apache/2.4.43 (Unix)
php_version: 7.4.30 64bit
php_sapi: fpm-fcgi
max_input_variables: 4000
time_limit: 1200
memory_limit: 256M
max_input_time: 600
upload_max_filesize: 300M
php_post_max_size: 1000M
curl_version: 7.84.0 (SecureTransport) OpenSSL/1.1.1q
suhosin: false
imagick_availability: false
pretty_permalinks: true
htaccess_extra_rules: false

### wp-database ###

extension: mysqli
server_version: 5.7.28
client_version: mysqlnd 7.4.30
max_allowed_packet: 16777216
max_connections: 151

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /Users/jangruchow/Local Sites/gamingpods/app/public/wp-content
WP_PLUGIN_DIR: /Users/jangruchow/Local Sites/gamingpods/app/public/wp-content/plugins
WP_MEMORY_LIMIT: 40M
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_ENVIRONMENT_TYPE: local
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

### pods ###

pods-server-software: Apache/2.4.43 (Unix)
pods-user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36
pods-session-save-path: undefined
pods-session-save-path-exists: No
pods-session-save-path-writable: No
pods-session-max-lifetime: 1440
pods-opcode-cache-apc: No
pods-opcode-cache-memcached: No
pods-opcode-cache-opcache: No
pods-opcode-cache-redis: No
pods-object-cache-apc: No
pods-object-cache-apcu: No
pods-object-cache-memcache: No
pods-object-cache-memcached: No
pods-object-cache-redis: No
pods-memory-current-usage: 48,402M
pods-memory-current-usage-real: 6,500M
pods-network-wide: No
pods-install-location: /wp-content/plugins/pods/
pods-developer: No
pods-tableless-mode: No
pods-relationship-table-enabled: Yes
pods-light-mode: No
pods-strict: No
pods-allow-deprecated: Yes
pods-api-cache: Yes
pods-shortcode-allow-evaluate-tags: No
pods-sessions: Disable sessions
pods-can-use-sessions: No
pods-relationship-table-status: 4

`
```

### Pods Package

```json
{
    "@meta": {
        "version": "2.9.8",
        "build": 1666003873
    },
    "settings": {
        "types_only": "0",
        "watch_changed_fields": "0",
        "metadata_integration": "1",
        "metadata_override_get": "0",
        "session_auto_start": "0",
        "wisdom_opt_out": "1"
    },
    "pods": [
        {
            "name": "post",
            "id": 5,
            "label": "Post",
            "description": "",
            "type": "post_type",
            "storage": "meta",
            "object": "post",
            "_migrated_28": "1",
            "groups": [
                {
                    "name": "more_fields",
                    "id": 6,
                    "label": "More Fields",
                    "description": "",
                    "weight": 0,
                    "fields": []
                }
            ]
        },
        {
            "name": "comparison",
            "id": 27,
            "label": "Vergleiche",
            "description": "",
            "type": "pod",
            "storage": "table",
            "label_singular": "Vergleich",
            "public": "1",
            "show_ui": "1",
            "pod_index": "name",
            "_migrated_28": "1",
            "detail_url": "vergleich\/{@permalink}\/",
            "hierarchical": "0",
            "pod_parent": "parent",
            "ui_style": "post_type",
            "show_in_menu": "1",
            "menu_position": "2",
            "menu_icon": "dashicons-star-empty",
            "ui_actions_enabled": [
                "add",
                "edit",
                "duplicate",
                "delete"
            ],
            "ui_fields_manage": [
                "id",
                "name",
                "created"
            ],
            "ui_filters": [
                "name"
            ],
            "required": "0",
            "unique": "0",
            "groups": [
                {
                    "name": "details",
                    "id": 28,
                    "label": "Details",
                    "description": "",
                    "weight": 0,
                    "fields": [
                        {
                            "name": "name",
                            "id": 29,
                            "label": "Name",
                            "description": "",
                            "weight": 0,
                            "type": "text",
                            "required": "1"
                        },
                        {
                            "name": "created",
                            "id": 30,
                            "label": "Date Created",
                            "description": "",
                            "weight": 1,
                            "type": "datetime",
                            "datetime_format": "ymd_slash",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mm_ss_A"
                        },
                        {
                            "name": "modified",
                            "id": 31,
                            "label": "Date Modified",
                            "description": "",
                            "weight": 2,
                            "type": "datetime",
                            "datetime_format": "ymd_slash",
                            "datetime_time_type": "12",
                            "datetime_time_format": "h_mm_ss_A"
                        },
                        {
                            "name": "author",
                            "id": 32,
                            "label": "Author",
                            "description": "",
                            "weight": 3,
                            "type": "pick",
                            "pick_object": "user",
                            "pick_format_type": "single",
                            "pick_format_single": "autocomplete",
                            "default_value": "{@user.ID}"
                        },
                        {
                            "name": "permalink",
                            "id": 33,
                            "label": "Permalink",
                            "description": "Leave blank to auto-generate from Name",
                            "weight": 4,
                            "type": "slug"
                        },
                        {
                            "name": "child_comparison",
                            "id": 35,
                            "label": "Child Comparison",
                            "description": "",
                            "weight": 5,
                            "type": "pick",
                            "pick_object": "pod",
                            "pick_val": "comparison",
                            "sister_id": "36",
                            "pick_table": "-- Select One --",
                            "required": "0",
                            "unique": "0",
                            "pick_format_type": "multi",
                            "pick_format_single": "dropdown",
                            "pick_format_multi": "list",
                            "pick_display_format_multi": "default",
                            "pick_display_format_separator": ", ",
                            "pick_allow_add_new": "1",
                            "pick_taggable": "0",
                            "pick_show_icon": "1",
                            "pick_show_edit_link": "1",
                            "pick_show_view_link": "1",
                            "pick_limit": "0",
                            "pick_user_role": "Administrator",
                            "pick_post_status": "publish",
                            "pick_post_author": "0",
                            "repeatable": "0",
                            "repeatable_format": "default",
                            "roles_allowed": "administrator",
                            "pick_display": "{@name}"
                        },
                        {
                            "name": "parent_comparison",
                            "id": 36,
                            "label": "Parent comparison",
                            "description": "",
                            "weight": 6,
                            "type": "pick",
                            "pick_object": "pod",
                            "pick_val": "comparison",
                            "sister_id": "35",
                            "pick_table": "-- Select One --",
                            "required": "0",
                            "unique": "0",
                            "pick_format_type": "multi",
                            "pick_format_single": "dropdown",
                            "pick_format_multi": "list",
                            "pick_display_format_multi": "default",
                            "pick_display_format_separator": ", ",
                            "pick_allow_add_new": "1",
                            "pick_taggable": "0",
                            "pick_show_icon": "1",
                            "pick_show_edit_link": "1",
                            "pick_show_view_link": "1",
                            "pick_limit": "0",
                            "pick_user_role": "Administrator",
                            "pick_post_status": "publish",
                            "pick_post_author": "0",
                            "repeatable": "0",
                            "repeatable_format": "default",
                            "roles_allowed": "administrator",
                            "pick_display": "{@name}"
                        },
                        {
                            "name": "views",
                            "id": 34,
                            "label": "views",
                            "description": "",
                            "weight": 7,
                            "type": "number",
                            "sister_id": "-- Select One --",
                            "required": "0",
                            "unique": "0",
                            "number_format_type": "number",
                            "number_format": "i18n",
                            "number_decimals": "0",
                            "number_format_soft": "0",
                            "number_step": "1",
                            "number_max_length": "12",
                            "number_html5": "0",
                            "repeatable": "0",
                            "repeatable_format": "default",
                            "roles_allowed": [
                                "administrator",
                                "editor"
                            ],
                            "admin_only": "0",
                            "restrict_role": "1",
                            "restrict_capability": "0"
                        }
                    ]
                }
            ]
        }
    ],
    "pages": [
        {
            "name": "asdfasDFASDF",
            "code": "",
            "id": 43,
            "_edit_last": "1",
            "_edit_lock": "1666003598:1"
        },
        {
            "name": "sdfsdf",
            "code": "",
            "id": 40,
            "_edit_last": "1",
            "_edit_lock": "1666003445:1"
        }
    ]
}
```

**Comment by sc0ttkclark** [comment]
Reproduced and looking into this

**Comment by sc0ttkclark** [comment]
Fixed via a4c62c7249212fc9e544b703415def6ac7299198 and 8a6198e6ef462f1f9c0d9b5572b9f119819102b5
